Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/Model/Game.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace NoughtsAndCrossesWPF
+{
+    class Game
+    {
+        public IBoardCommunication interpreter;
+        GameStatus gameStatus;
+        public Game(IBoardCommunication interpreter)
+        {
+            this.interpreter = interpreter;
+        }
+
+        public void GameActions(Player currentPlayer, GameBoard board, object field)
+        {
+            MakeMove(board, currentPlayer, field);
+            interpreter.ShowBoard(board);
+            gameStatus = SetGameStatus(board, currentPlayer);
+            if (gameStatus != GameStatus.Active) 
+            {
+                interpreter.ShowGameResult(board, currentPlayer, gameStatus); 
+            }           
+        }
+
+        public Player ChangePlayer(Player currentPlayer, Player player1, Player player2)
+        {
+            if (currentPlayer == player1)
+            {
+                return player2;
+            }
+            else
+            {
+                return player1;
+            }
+        }
+
+        public void MakeMove(GameBoard board, Player player, object field)
+        {
+            Position coordinates = interpreter.GetPlayerInput(field);
+            if (!board.CoordinatesAreValid(coordinates))
+            {
+                coordinates = interpreter.GetPlayerInput(field);
```

Comment:
MakeMove should take coordinates, then your GetPlayerInput() method can be somewhere else different depending on whether it's a console/WPF app